.github/workflows/license.yml

```diff
@@ -0,0 +1,52 @@
+name: V2 Licenses
+
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron: "23 18 * * *"
```

Comment:
Why "At 06:23 PM"? :)

Reply:
was following MLServer: https://github.com/SeldonIO/MLServer/blob/0b88ef73b6066663a4b074c20f9bb3a4c1dac0a2/.github/workflows/licenses.yml#L6 

but yeah I will change to perhaps midnight

Reply:
done in [deb2c9f](https://github.com/SeldonIO/seldon-core/pull/5465/commits/deb2c9fd30b5a99a0b36aa764387e21d3ea8b944)